mlir/lib/Conversion/TosaToTensor/TosaToTensor.cpp

```diff
@@ -268,12 +268,28 @@ class SliceConverter : public OpConversionPattern<tosa::SliceOp> {
     ShapedType resultType = cast<ShapedType>(sliceOp.getType());
     if (llvm::isa<UnrankedTensorType>(resultType))
       return failure();
+
+    ElementsAttr StartElems;
```

Comment:
Though we use `camelCase` in general.

Reply:
cc @tatwaichong suggests to conform to LLVM standard.

Reply:
Yes, it seems that we use `camelCase` in general. But I'm confused that should I start to follow the offical naming convention on my code.

As you knownm
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly
| Variable names should be nouns (as they represent state). The name should be camel case, and start with an upper case letter (e.g. Leader or Boats).

and "Our long term goal is for the entire codebase to follow the convention, ..." is mentioned in the document.

Honestly I'm not sure if this goal is still valid. I don't have strong opinion. It is super easy to change `CamelCase` to `camelCase`. Free feel to let me know if you think using camelCase naming is better in this case.

Reply:
Suggest to follow the MLIR style which deviates slightly from LLVM.